tests/Unit/Pypp/Pypp.hpp

```diff
@@ -90,6 +107,26 @@ struct SliceContainerImpl {
   }
 };
 
+// scalars are the only sort of spin-weighted type we support.
+template <typename T, int S>
+struct SliceContainerImpl<Scalar<SpinWeighted<T, S>>> {
+  static auto apply(const Scalar<SpinWeighted<T, S>>& container_dv,
+                    const size_t slice_idx) noexcept {
+    convert_to_container_of_doubles_t<std::decay_t<Scalar<T>>>
```

Comment:
Nitpick: I could be wrong but I don't think std::decay_t is needed here since Scalar is a concrete type.